pkg/descheduler/strategies/topologyspreadyconstraint.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
TODO: Add UT once we reach consensus on the api

Reply:
Change it to 2019..